extensions-core/azure-extensions/src/main/java/org/apache/druid/storage/azure/output/AzureStorageConnector.java

```diff
@@ -50,7 +51,7 @@ public class AzureStorageConnector extends ChunkingStorageConnector<AzureInputRa
 
   public AzureStorageConnector(
       final AzureOutputConfig config,
-      final AzureStorage azureStorage
+      @Global final AzureStorage azureStorage
```

Comment:
I dont think this annotation does anything without the constructor being annotated with `@inject`

Reply:
yeah its only used for msqe intermediate storage